src/node_sqlite.cc

```diff
@@ -1843,24 +1885,34 @@ void StatementSync::Get(const FunctionCallbackInfo<Value>& args) {
     return;
   }
 
-  LocalVector<Name> keys(isolate);
-  keys.reserve(num_cols);
-  LocalVector<Value> values(isolate);
-  values.reserve(num_cols);
+  if (stmt->return_arrays_) {
+    Local<Array> result = Array::New(isolate, num_cols);
```

Comment:
My understanding is that using a `LocalVector` is supposed to be more performant for cases like this. Can you update this to use that approach (similar to what is used below).